### DISCUSSION TITLE
Use child tags inside of a v-html Vue tag

### DISCUSSION THREAD

**Discussion by vdanviel** [community/community#119854] [Repositories]
### Select Topic Area

Show & Tell

### Body

well, i have this Vue.js project _[vdanviel/imaginenote](https://github.com/vdanviel/imaginenote)_ 

and i need to solve the following problem:

i need to be able to use child tags inside of a tag that contains the v-html attribute.
i've already added the necessary rules to desactivate the rule that forbidden me to do so `"vue/no-child-content"`, but somehow my system is not recognizing it.

my `.eslintrc.cjs`:

```
/* eslint-env node */
module.exports = {
  root: true,
  "extends": [
    "plugin:vue/vue3-strongly-recommended",
    "eslint:recommended"
  ],
  parserOptions: {
    ecmaVersion: "latest"
  },
  rules: {
    "vue/multi-word-component-names": 0,
    "vue/no-child-content": 0
  }
}
```

the source of the problem is on the `Note.vue` page on the project at `src/views/note/Note.vue`.

on this piece of the code:

```
    /* eslint vue/no-child-content: "off" */
    <draggable v-model="midia" tag="section">
      <template #item="{element: midia}">
        <div contenteditable="true" class="border resize-none rounded-lg px-4 py-2 flex-grow mb-[30px]" @input="update_text" @change="update_text" id="current_text" v-html="db_text">
          <Image :img_name="midia.name" :img_src="midia.src"/>
        </div>
      </template>
    </draggable>
```

do i have to some pre procedure so the eslint works? (i've already restart teh server and still not works)

thanks for the help.

**Comment by marioprieta** [comment]
The v-html directive in Vue.js is used to render raw HTML content within a component. However, it overrides any child elements inside the tag where it is applied. This is why you are encountering issues when trying to include child tags inside an element with v-html.

You've already attempted to disable the ESLint rule vue/no-child-content, which is correct for allowing child content within elements using directives like v-html. However, if this isn't working as expected, here are some steps to try to fix the issue:

1. Ensure Correct Rule Configuration: Double-check your ESLint configuration to ensure that the rule is correctly disabled. In your .eslintrc.cjs, you have:
 
```
rules: {
  "vue/multi-word-component-names": 0,
  "vue/no-child-content": 0
}
```

This should effectively disable the rule. Make sure there are no other configurations or overrides elsewhere in your project that might be re-enabling it.

2. Restart Your Development Environment: Sometimes changes in configuration files require a restart of your development server or IDE. Ensure that you have restarted both after making changes to .eslintrc.cjs.

3. Check for Other ESLint Configurations: If your project has multiple .eslintrc files (e.g., in subdirectories), ensure that none of them are re-enabling the rule.

4. Use Comments for Specific Cases: If disabling the rule globally doesn't work or isn't desirable, you can disable it for specific lines or files using comments:
 
```
<!-- eslint-disable-next-line vue/no-child-content -->
<div v-html="db_text">
  <Image :img_name="midia.name" :img_src="midia.src"/>
</div>
```
